src/Tribe/Views/V2/View.php

```diff
@@ -1349,4 +1369,32 @@ protected function setup_messages( array $events ) {
 			}
 		}
 	}
+
+	/**
+	 * Returns whether the View page should be reset or not.
+	 *
+	 * The View page should be reset when the View or filtering parameters that are not the page change.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool Whether the View page should be reset or not.
+	 */
+	protected function should_reset_page() {
+		if ( null === $this->should_reset_page ) {
+			$prev_url    = $this->context->get( 'view_prev_url', '' );
+			$current_url = $this->context->get( 'view_url', '' );
+
+			$view_data = $this->context->get( 'view_data', [] );
+			$bar_data = array_filter( $view_data, static function ( $value, $key ) {
```

Comment:
* :warning: Equals sign not aligned with surrounding assignments; expected 2 spaces but found 1 space
* :no_entry_sign: Opening parenthesis of a multi-line function call must be the last content on the line
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call

<sub>hash: 23dbad05b6871ce814de6c4706fdeac0</sub>